_redirects

```diff
@@ -505,6 +505,9 @@
 /v1.6/* https://docs.tigera.io/archive/v1.6/:splat 301
 /v1.5/* https://docs.tigera.io/archive/v1.5/:splat 301
 
+# Redirect to the root folder of Calico Eneterprise
```

Comment:
We can leave out the comment here. It's actually just a directory that grouped information about CE, but in the OS docs.